ml/notebooks/netdata_anomaly_detection_deepdive.ipynb

```diff
@@ -0,0 +1,1705 @@
+{
```

Comment:
<p>Now we are ready to loop over each row of data and produce anomaly scores. Once we have some trained models, <code>train_every</code> sets the frequency in which to (re)train or retrain the model.  </p><p> </p><p><strong>Note</strong>: The Netdata Agent implementation spreads out the training across each <code>train_every</code> window as opposed to trying to train all models in one go like the below implementation. It also avoids some obvious edges cases where there is no need to retrain, for example when the data have not changed at all since last model was trained.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netdata/netdata/pull/12313/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>